### PR TITLE
Metrics Updates

### PR DESCRIPTION
**Pull request by ghickman**
This updates the Metrics list page to be inline with the rest of the sites list pages and includes Metrics in the search.

### CODE REVIEW

**Comment by fredkingham** [comment]
Can we urlise business definition and source address. 
Can we not show the well in technical specification if its empty.

/ncdr/metrics/176/

**Comment by fredkingham** [comment]
All wells in the list view state...

Screen Shot 2019-03-15 at 10 43 06

And the code states
```
    <div class="row nhs-england-well">
      <div class="col-md-12">
        <p>
          TODO METRICS
        </p>
      </div>
    </div
```

Is this a TODO for us? Are we waiting on data?

**Comment by fredkingham** [comment]
Can we do something with putting the tsv's into the repo? I'd like to try it locally.

Or are we thinking their loading is a seperate PR?

**Comment by ghickman** [comment]
> Can we urlise business definition and source address.

Done in 7c5c356

Can we not show the well in technical specification if its empty.

Done in a8493da

---

> Is this a TODO for us? Are we waiting on data?

Yes, I didn't know what to put there.  On other list pages we have things like:

    Data elements are collections of similarly defined data items, potentially from different sources.

---

> Can we do something with putting the tsv's into the repo? I'd like to try it locally.

I've added the TSVs under `data/metrics` and `make metrics` to simplify that process for you.  I don't know what the situation is with adding them as I'm not sure where the data is coming from in the future so lets make that call when we can answer that.

File: metrics/management/commands/import_themes.py [hunk 1]
```diff
@@ -16,5 +16,8 @@ def handle(self, *args, **options):
             delimiter = "\t" if path.endswith(".tsv") else ","
             rows = list(csv.DictReader(f, delimiter=delimiter, quotechar='"'))
 
-        for row in rows:
-            Theme.objects.create(number=row["Number"], name=row["Theme"])
+        Theme.objecs.bulk_create(
```
Comment (fredkingham) [inline metrics/management/commands/import_themes.py]: typo?

**Comment by fredkingham** [comment]
can we fix the breadcrumbs to use the name of the metric

Screen Shot 2019-03-18 at 08 51 53

File: metrics/management/commands/import_themes.py (hunk 1 above)
Comment (ghickman) [reply metrics/management/commands/import_themes.py]: Oops, yes, fixed and rebased.

**Pull request merged**